src/main/java/thaumic/tinkerer/common/block/tile/TileCamo.java

```diff
@@ -18,7 +18,9 @@
 import net.minecraft.tileentity.TileEntity;
 
 import appeng.api.movable.IMovableTile;
+import cpw.mods.fml.common.Optional;
 
+@Optional.Interface(iface = "appeng.api.movable.IMovableTile", modid = "appliedenergistics2")
```

Comment:
Last comment (hopefully): You also need to mark any method you override from this interface as `@Optional.Method(modid = "appliedenergistics2")`